django/db/models/query.py

```diff
@@ -301,13 +302,19 @@ def __or__(self, other):
     # METHODS THAT DO DATABASE QUERIES #
     ####################################
 
-    def iterator(self):
+    def iterator(self, chunk_size=2000):
         """
         An iterator over the results from applying this QuerySet to the
         database.
         """
-        use_chunked_fetch = not connections[self.db].settings_dict.get('DISABLE_SERVER_SIDE_CURSORS')
-        return iter(self._iterable_class(self, chunked_fetch=use_chunked_fetch))
+        if chunk_size is None:
+            use_chunked_fetch = False
+            chunk_size = 2000
```

Comment:
This value has a significance for the database adapter, it determines how many result rows are transformed to Python objects per batch. For example, [see this psycopg `for` loop](https://github.com/psycopg/psycopg2/blob/de8b335d80ecba0305e2b4796373a4064fd450b3/psycopg/cursor_type.c#L958-L965).

I don't think it matters much to most users though. Since the beginning of project git history, Django has always used `GET_ITERATOR_CHUNK_SIZE` (100) for fetchmany.